proto/wippersnapper/uart/v1/uart.proto

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2023 Brent Rubell for Adafruit Industries
+// SPDX-License-Identifier: MIT
+syntax = "proto3";
+
+package wippersnapper.uart.v1;
+import "nanopb/nanopb.proto";
+import "wippersnapper/i2c/v1/i2c.proto";
+
+/**
+* UARTBusData represents a message to configure a UART bus for communication with a device.
+* NOTE: This message is never sent directly, it is packed inside UARTDeviceAttachRequest.
+*/
+message UARTBusData {
+  int32 baudrate        = 1; /** The baudrate to use for UART communication (may be a common baud rate such as: 1200bps, 2400bps, 4800bps, 19200bps, 38400bps, 57600bps, or 115200bps). */
+  int32 pin_rx         = 2; /** The pin on which to receive UART stream data. */
+  int32 pin_tx         = 3; /** The pin on which to transmit UART stream data. */
```

Comment:
Comparing with `pin.proto`, why is this one using `int32` for the pin when others use the `string` name?

Reply:
It should use the `string` name, corrected in next commit